File: app/src/main/java/com/leanima/database/ItemViewHolder.java
package com.leanima.database;

import android.view.View;
import android.widget.ImageView;
import android.widget.TextView;
import androidx.annotation.NonNull;
import androidx.recyclerview.widget.RecyclerView;

import com.leanima.R;

public class ItemViewHolder extends RecyclerView.ViewHolder {
    final TextView itemName;
    final TextView itemPrice;
    final ImageView itemImage;

    public ItemViewHolder(@NonNull View itemView) {
        super(itemView);
        itemName = itemView.findViewById(R.id.itemName);
        itemPrice = itemView.findViewById(R.id.itemPrice);
        itemImage = itemView.findViewById(R.id.itemImage);
    }
}

File: app/src/main/java/com/leanima/docker/php/www/ItemViewHolder.kt
import android.view.View
import android.widget.ImageView
import android.widget.TextView
import androidx.recyclerview.widget.RecyclerView
import com.leanima.R

class ItemViewHolder(view: View) : RecyclerView.ViewHolder(view) {
    val itemImage: ImageView = view.findViewById(R.id.item_image)
    val itemName: TextView = view.findViewById(R.id.item_name)
    val itemPrice: TextView = view.findViewById(R.id.item_price)
}
